Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:16-alpine as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm ci
+# RUN $(npm bin)/ncc build -m src/Application.ts
+RUN npm run build
```

Comment:
Hey there, thanks for your PR. One Thing though, everytime when i change something i also update the index so its kinda pointless to rebuild the whole bot on every start

Reply:
These are build time commands, which are only run when a container is being built. The only commands run on startup are the 2 final lines (`ENTRYPOINT` and `CMD`).

Even if you update the `index.js` every time, this will still guarantee that the file is up to date within the container. That being said, the extra comment is something I will remove.